.gitignore

```diff
@@ -21,7 +21,7 @@
 **/.vscode/settings.json
 **/create-stack.sh
 **/update-packages.sh
-**/package-lock.json
```

Comment:
I guess this would cause to create `package-lock.json` files which haven't been tracked by the repo. Whenever someone clone the repository and build the solution, I think it would create `package-lock.json` files as new files.

Reply:
The way build script is written, it is deleting all package-locks for lambda functions, but not for utils.
Lately, we  have been following the practice of checking in package-lock files and that is why I removed the gitignore for it.